tests/cecil-tests/GetterExceptionTest.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using Mono.Cecil;
+using System.Collections.Generic;
+using NUnit.Framework;
+using System.Linq;
+using Mono.Cecil.Cil;
+using Xamarin.Utils;
+using Xamarin.Tests;
+
+namespace Cecil.Tests {
+	[TestFixture]
+	public class GetterExceptionTest {
+		Dictionary<string, AssemblyDefinition> assemblyCache = new ();
```

Comment:
```suggestion
		Dictionary<string, AssemblyDefinition> assemblyCache = new Dictionary<string, AssemblyDefinition> ();
```

Reply:
Using new this way is new (pun not intended) to C# 9.0: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/new-operator#:~:text=20%3B%20third%3A%2030-,Beginning,-with%20C%23%209.0. 

It's a way to avoid using the var keyword (which can be a touchy topic lol). If it doesn't fit the code style for the team though, I can use var or keep the verbosity.